### PR TITLE
Added coverage for LDAP in CLI

### PR DESCRIPTION
**Pull request by oshtaier**
Just coverage as stupid me misunderstood required server type in https://bugzilla.redhat.com/show_bug.cgi?id=1293538

```
nosetests tests/foreman/cli/test_ldapauthsource.py
.
----------------------------------------------------------------------
Ran 1 test in 77.345s

OK
```
```
nosetests tests/foreman/cli/test_usergroup.py:ActiveDirectoryUserGroupTestCase
.
----------------------------------------------------------------------
Ran 1 test in 32.682s

OK
```

### CODE REVIEW

File: tests/foreman/cli/test_usergroup.py [hunk 1]
```diff
@@ -648,3 +658,50 @@ def test_positive_remove_usergroup_by_name(self):
...
+        :expectedresults: User group is created successfully and assigned to
+            correct auth source
+
+        :CaseLevel: Integration
+        """
+        user_group = make_usergroup()
+        ext_user_group = make_usergroup_external({
+            'auth-source-id': self.auth['id'],
+            'user-group-id': user_group['id'],
+            'name': 'foobargroup'
```
Comment (svtkachenko) [inline tests/foreman/cli/test_usergroup.py]: Why not gen_string? :)

Comment (oshtaier) [inline tests/foreman/cli/test_usergroup.py]: because it is user group name in Active Directory =)

Comment (svtkachenko) [inline tests/foreman/cli/test_usergroup.py]: Ah, indeed. :)

File: tests/foreman/cli/test_ldapauthsource.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+from robottelo.decorators import skip_if_not_set, tier1
+from robottelo.test import CLITestCase
+
+
+class LDAPAuthSourceTestCase(CLITestCase):
+    """Implements Active Directory feature tests in CLI"""
+
+    @classmethod
+    @skip_if_not_set('ldap')
+    def setUpClass(cls):
```
Comment (abalakh) [inline tests/foreman/cli/test_ldapauthsource.py]: Add a docstring, please

File: tests/foreman/cli/test_usergroup.py [hunk 3]
```diff
@@ -648,3 +658,50 @@ def test_positive_remove_usergroup_by_name(self):
...
         user_group = UserGroup.info({'id': user_group['id']})
         self.assertEqual(len(user_group['user-groups']), 0)
+
+
+class ActiveDirectoryUserGroupTestCase(CLITestCase):
+    """Implements Active Directory feature tests for user groups in CLI."""
+
+    @classmethod
+    @skip_if_not_set('ldap')
+    def setUpClass(cls):
```
Comment (abalakh) [inline tests/foreman/cli/test_usergroup.py]: same here

**Pull request merged**